test/integration/custom-routes/test/index.test.js

```diff
@@ -278,13 +278,15 @@ const runTests = (isDev = false) => {
             statusCode: 307,
             regex: normalizeRegEx('^\\/hello(?:\\/([^\\/]+?))\\/another$'),
             regexKeys: ['id'],
+            permanent: false,
```

Comment:
Isn't this not needed because statusCode is generated by the framework.

Reply:
Right we shouldn't need it, updated to not output it